lib/libzfs/libzfs_util.c

```diff
@@ -46,6 +46,18 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 
+#ifdef HAVE_CURSES_COLOR
+#include <curses.h>
+#include <term.h>
+/*
+ * For some reason, term.h #defines 'lines' (in lower case) as a macro.
+ * Since we also have a variable named 'lines', undefine the macro here.
```

Comment:
Even though this works, I think we should go ahead and rename the our local `lines` variable to avoid the conflict entirely.  Then drop this block.  According to the curses documentation, `curs_terminfo(3X)`.

```
      ·   If use_env(FALSE) has been called, values for lines and  columns  specified
           in terminfo are used.
```

Which would be handy if we decide it's desirable to use more of the libraries functionality.